README.md

```diff
@@ -9,24 +9,33 @@ This site is written using [Jekyll](https://jekyllrb.com/), a static site genera
 ## Local Development
 
 #### Prerequisites:
-1. [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
-2. [Bundler](https://bundler.io/) (`gem install bundler`)
+
+1. [Docker](https://docker.com) or [Podman](https://podman.io)
 
 #### Instructions
-1. `bundle install`
-2. Add necessary environment variables (they MUST be defined!!!):
-	* `export CSHPUBSITE_ASSETS_URL="https://assets.csh.rit.edu/pubsite"`
-	* `export CSHPUBSITE_S3_URL="https://s3.csh.rit.edu"`
-3. `bundle exec jekyll serve`
 
-The site should now serve locally at `localhost:4000`, and auto-build when you change/create files in the repo!
```

Comment:
I'd say this is a significant advantage of keeping the local build option around. Iterating on a docker container is somewhat slow

Reply:
That is fair, I personally had issues running it locally, as it would error when I ran it even after following the instructions exactly, but maybe that is just a me issue. If it is just a me issue, I can readd the info about running locally without a container.

Reply:
Might be a build issue, but I think keeping both in docs can be valuable

Reply:
I updated the instructions to have both ways